### ISSUE TITLE
Cross-check accounts with EveryPolitician?

### ISSUE THREAD

**Issue by briatte** [briatte/elus#5]
@florence-nocca suggests that we take a look at the data collected by @tmtmtmtm and others at [EveryPolitician](http://everypolitician.org/), to see whether it can be useful to improve our own (French) politicians data

EveryPolitician has data for terms 12, 13 and 14 of the National Assembly:

- https://github.com/everypolitician/everypolitician-data
  - [tree/master/data/France](https://github.com/everypolitician/everypolitician-data/tree/master/data/France)
  - [tree/master/data/France/National_Assembly](https://github.com/everypolitician/everypolitician-data/)
- cross-check gender, mandates?
- cross-check Twitter handles?

**Comment by tmtmtmtm** [comment]
Hi @briatte — this certainly looks like an interesting project! We've been planning to significantly improve our French data for a while now (including adding a lot more historical data), and this might be a useful impetus for that! So if there's anything we can do that would be useful for you (or if you spot any problems in our existing data), please do let us know.

**Comment by briatte** [comment]
Thanks @tmtmtmtm – the project in this repo is about Twitter account holders among French politicians, but I must have told you about the [Sycomore](http://www2.assemblee-nationale.fr/sycomore/recherche) historical database of French MPs in the past, right?

**Comment by tmtmtmtm** [comment]
Yes: there's an initial stab at a scraper at https://github.com/everypolitician-scrapers/france-national-assembly-sycomore, but if I remember correctly there were more issues with the data from there than I had expected, particularly around dates. This was a while ago, though, so the details are a little hazy. I think that between Sycomore, Wikidata, and the French Wikipedia, there's almost certainly very good data now that goes back to at least 1958. We just need to get around to tying it all together.

**Comment by briatte** [comment]
I barely remember [my own code](https://github.com/briatte/parlement/blob/master/sponsors-an.r), but yes, there are issues with the data, perhaps especially if you want precise (yyyy-mm-dd) dates of entry/exit to the lower chamber. Things are easier if you just need the legislature number, which is my case.

Tying it together would be nice indeed, but that's a real technical challenge. It's on my TODO list for the summer, when I start updating [`parlnet`](https://github.com/briatte/parlnet) again.
